graphql/schemas/Social/channels.graphql

```diff
@@ -0,0 +1,67 @@
+extend type User {
+    pivot: SocialChannelPivot
+}
+
+
+type SocialChannel {
+    id: ID!
+    name: String!
+    slug: String
+    description: String!
+    entity_namespace: String!
+    entity_id: ID!
+    last_message_id: ID!
+    messages: [Message!]! @belongsToMany
+    users: [User] @belongsToMany
+    systemModule: SystemModule!
+}
+
+input SocialChannelInput {
+    name: String!
+    slug: String
+    description: String!
+    entity_namespace_uuid: String!
+    entity_id: ID!
+}
+
+enum UsersRolesChannel{
+    Admin @enum(value: 1)
+    User @enum(value: 2)
+}
+
+type SocialChannelPivot {
+    roles_id: UsersRolesChannel!
+}
+
+
+extend type Query {
+    getSocialChannels(
+        where: _@whereConditions(
+            columns: [
+                "id",
+                "name",
+                "slug",
+                "description",
+                "entity_namespace",
+                "entity_id",
+            ]
+        )
+    ): [SocialChannel] @paginate(
```

Comment:
@FredPeal we need to add company_id to the channel , so we can filter the content and avoid leaking other company info if they guess the id